format_strawberryfield.libraries.yml

```diff
@@ -136,13 +136,13 @@ pdfs_strawberry:
     - format_strawberryfield/pdfs_mozilla
 
 mirador_projectmirador:
-  version: 3.0.0-rc.5
+  version: 3.0.0
   license:
     name: Apache
     url: //github.com/ProjectMirador/mirador/blob/master/LICENSE
     gpl-compatible: true
   js:
-    https://cdn.jsdelivr.net/npm/mirador@3.0.0-rc.5/dist/mirador.min.js: { external: true, minified: true, preprocess: false}
+    https://cdn.jsdelivr.net/npm/mirador@3.0.0/dist/mirador.min.js: { external: true, minified: true, preprocess: false}
```

Comment:
This here is already in ISSUR-102 and its own issue . Can merge of course but I will need to rebase. Will let you know